### ISSUE TITLE
Where we can download or get the software

### ISSUE THREAD

**Issue by artayasa** [wiciu15/servo2#4]
Hi mate where we can get the software, i wan to try your code.

**Comment by wiciu15** [comment]
Clone the repo or use green download as zip button

**Comment by artayasa** [comment]
> Clone the repo or use green download as zip button

Yes but i dont see the software for setting like u sir. Its not open source??

**Comment by wiciu15** [comment]
I can't share PC application due to licensing of internal components. Communication to servo is modbus so any modbus analyzer would work similarly

**Comment by artayasa** [comment]
> I can't share PC application due to licensing of internal components. Communication to servo is modbus so any modbus analyzer would work similarly

Is posibl without setting from pc? So one time settig ithe code, wich code i can change?? Also my ipm module is active low, is posible to invert the output pwm HIN and LIN??.
